### PR TITLE
add weasyprint function

### PR DESCRIPTION
**Pull request by hiaselhans**
that was actually for free, but maybe should we do imports inside the function so there is not another dependency added?

(sorry for bombing so many pr's ;) )

### CODE REVIEW

File: report/__init__.py [hunk 1]
```diff
@@ -19,6 +20,7 @@
 from jinja2 import Environment, FunctionLoader
 from babel.dates import format_date, format_datetime
 from babel.numbers import format_currency
+import weasyprint
```
Comment (tarunbhardwaj) [inline report/__init__.py]: Lets not make this a requirement ?

```python

try:
    import weasyprint
except ImportError:
    weasyprint = None
```

File: setup.py [hunk 2]
```diff
@@ -46,6 +46,7 @@ def run(self):
     'executor',
     'jinja2',
     'babel',
+    'weasyprint'
```
Comment (tarunbhardwaj) [inline setup.py]: How about making this optional ?

```python
extras_require={
        'weasyprint': ['weasyprint'],
}
```

and then we could use ``pip install openlabs_report_webkit[weasyprint]``

**Comment by tarunbhardwaj** [comment]
Hey @hiaselhans thanks for the patch. I had a few comments. Given that we now have support outside of webkit, do you think we should rename this module ? perhaps `report_html` ? that seems to be the key - the ability to write reports in html

**Comment by hiaselhans** [comment]
hola @tarunbhardwaj! thanks for all the work with merges!
Good idea about making weasyprint optional, basically i just found weasyprint pretty performant (faster than pdfwebkit) and just put it in here because it wasnt too hard...
Maybe naming it report_html would be a good choice..! (maybe trytond_report_html to clarify) but then i think it should be a general feature to choose your rendering engine...
enough said, should i  add your suggestions  and squash commits?
regards
:sunny:

File: setup.py (hunk 2 above)
Comment (hiaselhans) [inline setup.py]: thats cool!

**Comment by sharoonthomas** [comment]
@hiaselhans I think you can send a new PR with squashed commit and the changes @tarunbhardwaj mentioned.

:+1:  thanks again for the contrib

**Comment by sharoonthomas** [comment]
#25 implements the changes suggested here and has been merged

**Pull request closed without merging**